game-core/src/main/java/games/strategy/triplea/attachments/TerritoryEffectAttachment.java

```diff
@@ -48,8 +49,10 @@ private void setCombatDefenseEffect(final String combatDefenseEffect) throws Gam
     setCombatEffect(combatDefenseEffect, true);
   }
 
-  private void setCombatDefenseEffect(final IntegerMap<UnitType> value) {
+  @VisibleForTesting
```

Comment:
It does not look like this is yet called from tests, did you mean to commit changes to this file as part of this PR?

Reply:
It is being used at https://github.com/triplea-game/triplea/pull/7891/files#diff-bf155c528a8a462543b7b54186929016c76294d8a0d9640f3f2357cf7d9da6b9R68.